bands/site_sections/band_admin.py

```diff
@@ -12,6 +12,9 @@ def index(self, session, message=''):
     @ajax
     def mark_as_band(self, session, group_id):
         group = session.group(group_id)
+        if not group.leader:
+            return {'message': '{} does not have an assigned group leader'.format(group.name)}
```

Comment:
Certain steps would error in the absence of a group leader, plus there would be no one to receive an email with the link to the page in the first place.